File: src/test/java/com/hyak4j/student/StudentTest.java
package com.hyak4j.student;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

public class StudentTest {

    @Test
    void highestTest(){
        final int english = 60;
        final int math = 80;
        Student student = new Student("Henry", english, math);
        Assertions.assertEquals(Math.max(english, math), student.highest());
    }

    @Test
    void averageTest(){
        final int english = 60;
        final int math = 80;
        Student student = new Student("Henry", english, math);
        Assertions.assertEquals((english + math) / 2, student.getAverage());
    }
}

File: src/test/kotlin/com/hyak4j/StudentTest.kt
package com.hyak4j

import com.hyak4j.kotlin.student.Student
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class StudentTest {
    val english = 60
    val math = 80

    @Test
    fun highestTest(){
        val student = Student("Herny", english, math)
        Assertions.assertEquals(math, student.highest())
    }

    @Test
    fun averageTest(){
        val student = Student("Herny", english, math)
        Assertions.assertEquals((english + math) / 2, student.getAverage())
    }

    @Test
    fun gradingTest(){
        val student = Student("Herny", english, math)
        Assertions.assertEquals('C', student.grading())
    }
}